jwst/regtest/test_miri_lrs_slit_spec2.py

```diff
@@ -33,7 +37,7 @@ def run_pipeline(jail, rtdata_module):
 
 @pytest.mark.bigdata
 @pytest.mark.parametrize("output",[
-    "bsub", "flat_field", "srctype", "cal", "x1d"])
+    "bsub", "flat_field", "assign_wcs","srctype", "cal", "x1d"])
```

Comment:
flake8 will complain about the lack of whitespace between "assign_wcs" and "srctype"